rpm/usbrelay.spec

```diff
@@ -126,7 +157,11 @@ touch %{buildroot}%{_rundir}/usbrelay/usbrelayd.pid
 %{_sysconfdir}/usbrelayd.conf
 %attr(0755,usbrelay,usbrelay) %ghost %dir %{_rundir}/usbrelay/
 %attr(0644,usbrelay,usbrelay) %ghost %{_rundir}/usbrelay/usbrelayd.pid
+%{_mandir}/man8/usbrelayd.8.gz
 
+%files devel
+%{_includedir}/libusbrelay.h
+%{_libdir}/libusbrelay.so
```

Comment:
I don't think devel packages should contain any `.so` files, even if they are just symlinks.